pom.xml

```diff
@@ -375,17 +375,33 @@
         </executions>
       </plugin>
 
-      <!-- Find Bugs -->
+      <!-- FindBugs -->
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>findbugs-maven-plugin</artifactId>
-        <version>2.5.2</version>
+        <version>3.0.2</version>
         <configuration>
-          <excludeFilterFile>${findbugs.path}findbugs-exclude.xml</excludeFilterFile>
+          <excludeFilterFile>${findbugs.path}/findbugs-exclude.xml</excludeFilterFile>
+          <includeFilterFile>${findbugs.path}/findbugs-include.xml</includeFilterFile>
+          <!--
+              Enables analysis which takes more memory but finds more bugs.
+              If you run out of memory, changes the value of the effort element
+              to 'Low'.
+            -->
+          <effort>Max</effort>
+          <failOnError>true</failOnError>
+          <threshold>Low</threshold>
           <findbugsXmlOutput>true</findbugsXmlOutput>
           <xmlOutput>true</xmlOutput>
-          <effort>Max</effort>
         </configuration>
+        <executions>
+          <execution>
+            <phase>verify</phase>
```

Comment:
Done, these are probably more likely to fail than the tests (once we unsuppress more warnings at least), and take less time.